apps/pre/pre-api/prod.yaml

```diff
@@ -7,4 +7,19 @@ spec:
   releaseName: pre-api
   values:
     java:
-      ingressHost: pre-api.platform.hmcts.net
\ No newline at end of file
+      ingressHost: pre-api.platform.hmcts.net
+      keyvaults:
+        "pre-hmctskv":
```

Comment:
you also need to disable the original key vault name:
https://github.com/hmcts/chart-library#example-for-adding-azure-key-vault-secrets

honestly it would be cleaner just to rename the non production vaults to match prod as otherwise this will be a maintenance nightmare, as prod secrets will be missed every time

Reply:
okay, will take this approach instead, thank you